src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -349,7 +349,10 @@ public static void register() {
                 .flags(EXT_METAL, GENERATE_SPRING)
                 .components(Nickel, 4, Chrome, 1)
                 .cableProperties(V[EV], 4, 4)
-                .blastTemp(2700, GasTier.LOW, VA[HV], 1300)
+                .blast(b -> b
+                        .temp(2700, GasTier.LOW)
+                        .blastStats(VA[EV], 1300)
```

Comment:
Was moving this to EV intentional?

Reply:
Yes tungstencarbide was intentionally raised, all other tungsten metals were EV so felt like a strange inconsistency